webserver/src/main/java/io/kestra/webserver/controllers/TriggerController.java

```diff
@@ -79,4 +80,134 @@ public HttpResponse<Trigger> unlock(
 
         return HttpResponse.ok(trigger);
     }
+
+    @ExecuteOn(TaskExecutors.IO)
+    @Get(uri = "/{namespace}/{flowId}")
+    @Operation(tags = {"Triggers"}, summary = "Get all triggers for a flow")
+    public PagedResults<Trigger> find(
+        @Parameter(description = "The current page") @QueryValue(defaultValue = "1") int page,
+        @Parameter(description = "The current page size") @QueryValue(defaultValue = "10") int size,
+        @Parameter(description = "The sort of current page") @Nullable @QueryValue List<String> sort,
+        @Parameter(description = "A string filter") @Nullable @QueryValue(value = "q") String query,
+        @Parameter(description = "A namespace filter prefix") @Nullable @QueryValue String namespace,
+        @Parameter(description = "A flow id") @Nullable @QueryValue String flowId
+    ) throws HttpStatusException {
+        return PagedResults.of(triggerRepository.find(
+            PageableUtils.from(page, size, sort, triggerRepository.sortMapping()),
+            query,
+            tenantService.resolveTenant(),
+            namespace,
+            flowId
+        ));
+    }
+
+    @ExecuteOn(TaskExecutors.IO)
+    @Put(uri = "/")
+    @Operation(tags = {"Triggers"}, summary = "Update a trigger")
+    public HttpResponse<Trigger> update(
+        @Parameter(description = "The trigger") @Body final Trigger trigger
+    ) throws HttpStatusException {
+        Trigger updatedTrigger = this.triggerRepository.lock(trigger.uid(), (current) -> {
+            Trigger updated = trigger;
+            if (trigger.getBackfill() != null) {
```

Comment:
This code should be inside Trigger.update(current, new).
It's strange to have some part of the trigger update here and the other there